packages/xs-extra/wsproxy.master/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "wsproxy"
+version: "1.1.0"
+authors: [ "Jon Ludlam" "jonathan.ludlam@citrix.com"
+           "Marcello Seri <marcello.seri@citrix.com>" ]
+license: "LGPL-2.0 with OCaml linking exception"
+homepage: "https://github.com/xapi-project/wsproxy"
+dev-repo: "https://github.com/xapi-project/wsproxy"
+bug-reports: "https://github.com/xapi-project/wsproxy/issues"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: [
+  ["ocaml" "setup.ml" "-install"]
+]
+remove: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-uninstall"]
+]
+depends: [
+  "lwt" {build}
```

Comment:
Oops, my bad, we should fix the `opam` files of `vncproxy` too, I think `build` should be used for things like `oasis`, but not the dependencies that are compiled into it, but I'm not sure though. Based on the above description, I think we do want it to be recompiled when one of the normal dependencies change, so we should probably drop `{build}` for all of these, except `ocamlbuild`, `ocamlfind`, and `oasis`.